docs/source/library-user-guide/adding-udfs.md

```diff
@@ -76,7 +76,8 @@ The challenge however is that DataFusion doesn't know about this function. We ne
 
 ### Registering a Scalar UDF
```

Comment:
Do we want to add advanced example `advanced_udf.rs` link to this document and also document `ScalarUDFImpl` there too? Maybe a follow up.